wordpress-mobile/WordPress-iOS
# Remove WPMediaPicker dependency

Removes WPMediaPicker dependency

To test:

- Smoke test Gutenberg: verify you can add images from Photos and Site Media
- Smoke test Site Media: verify that you can open video previews
- Smoke test Site Icon Picker: verify that picking from Site Media works

## Regression Notes
1. Potential unintended areas of impact: Media
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
